Px.Utils/Models/Metadata/Dimensions/ContentDimensionValue.cs

```diff
@@ -63,6 +65,30 @@ public ContentDimensionValue(
             Precision = precision;
         }
 
+
+        /// <summary>
+        /// Constructor mainly for deserialization purposes. 
+        /// </summary>
+        /// <param name="code">Unique code among the values of this dimension</param>
+        /// <param name="name">Multilanguage name of the dimension value</param>
+        /// <param name="isVirtual">Indicates whether the value is in the original data or was created by computing or aggregating other values</param>
+        /// <param name="additionalProperties">Additional properties of the dimension value</param>
+        [JsonConstructor]
+        public ContentDimensionValue(
+            string code,
+            MultilanguageString name,
+            MultilanguageString unit,
```

Comment:
unit, lastUpdated and precision are not documented in the summary params above

Reply:
Documentation added.